tests/unit/Tenancy/Database/AutoListenerTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the tenancy/tenancy package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see http://laravel-tenancy.com
+ * @see https://github.com/tenancy
+ */
+
+namespace Tenancy\Tests\Database;
+
+use Tenancy\Testing\TestCase;
+use Tenancy\Database\Events\Resolving;
+use Tenancy\Tenant\Events\Created;
+use Tenancy\Tenant\Events\Updated;
+use Tenancy\Tenant\Events\Deleted;
+
+class AutoListenerTest extends TestCase
+{
+    protected function register_db_driver()
```

Comment:
This is not a test method and as such should follow the naming conventions of a regular method: registerDatabaseDriver